lib/membrane_h264_format/remote.ex

```diff
@@ -1,14 +1,19 @@
 defmodule Membrane.H264.RemoteStream do
   @moduledoc """
-  Format definition for H264 video streams with decoder configuration transmitted out of band.
+  Format definition for packetized, remote H264 video streams.
 
-  Example of such a stream is H264 depayloaded from a container like MP4 or FLV, where decoder configuration is signalled outside of the H264 bytestream.
+  Examples of such a stream:
+  * H264 depayloaded from a container like FLV, where
```

Comment:
Removed reference to `MP4` as our MP4 plugin doesn't output H264.RemoteStream caps